### PR TITLE
Tests: Output Content Provider query related functionality 

### PR DESCRIPTION
**Pull request by Raymondd**
Addresses #584.
The request handler tests seem like they may need to be refactored to be properly testable. I created Issue #625 to track these changed since they are going to be significantly different from these query related tests.

### CODE REVIEW

File: test/webServiceRequestHandlers.test.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
+// TODO: rewrite all the outputprovider handle tests (old ones kept for reference)
```
Comment (kevcunnane) [inline test/webServiceRequestHandlers.test.ts]: I'd remove this test class if it's no longer relevant. People can go back in Git history if they need to retrieve at all. Unless there's a reason to preserve?

File: src/models/SqlOutputContentProvider.ts [hunk 2]
```diff
@@ -275,8 +275,12 @@ export class SqlOutputContentProvider implements vscode.TextDocumentContentProvi
         queryRunner.runQuery(selection);
         let paneTitle = Utils.formatString(Constants.titleResultsPane, queryRunner.title);
         // Always run this command even if just updating to avoid a bug - tfs 8686842
+        this.displayResultPane(resultsUri, paneTitle);
+    }
 
-        // Check if the results window already exists
+    // Intentionally created anonymously for testing purposes
+    public displayResultPane = function(resultsUri: string, paneTitle: string): void {
```
Comment (kevcunnane) [inline src/models/SqlOutputContentProvider.ts]: This is kinda goofy. I think using TypeMoq you can just mock a real instance of this class and override this one function, avoiding the need to add in a setDisplayResultPane and declare this anonymously. However it's not critical to fix.

File: test/webServiceRequestHandlers.test.ts (hunk 1 above)
Comment (Raymondd) [inline test/webServiceRequestHandlers.test.ts]: I implement this TODO in another PR. Should have probably created this file in that one, but I will send the other one in right after this one, so this shouldn't be a problem.

File: src/models/SqlOutputContentProvider.ts (hunk 2 above)
Comment (Raymondd) [inline src/models/SqlOutputContentProvider.ts]: I couldn't do a partial Moq, but I did fix this goofy syntax in the class.

**Pull request merged**